addons/l10n_in/views/account_account_views.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <record id="account_account_tds_tcs_view_form_inherit" model="ir.ui.view">
-        <field name="name">account.account.tds.tcs.view.form.inherit</field>
+    <record id="view_account_account_form_inherit_l10n_in" model="ir.ui.view">
+        <field name="name">account.account.form.inherit.l10n_in</field>
```

Comment:
don't change XML IDs else we need to add this in migration

Reply:
done